### PR TITLE
Add support for leafs 0x1609 and 0x160a

### PR DESCRIPTION
**Pull request by mahkoh**

### CODE REVIEW

File: src/tpi/data.rs [hunk 1]
```diff
@@ -92,6 +92,27 @@ pub(crate) fn parse_type_data<'t>(mut buf: &mut ParseBuffer<'t>) -> Result<TypeD
             Ok(TypeData::Class(class))
         }
 
+        // https://github.com/microsoft/microsoft-pdb/issues/50#issuecomment-737890766
+        LF_STRUCTURE19 => {
+            let mut class = ClassType {
+                kind: ClassKind::Struct,
+                properties: TypeProperties(buf.parse_u32()? as u16),
```
Comment (jan-auer) [inline src/tpi/data.rs:99]: If `properties` has been extended to `u32`, this suggests that there are more properties carried by the new type. Should we already extend the inner value of TypeProperties to `u32`?

**Review by jan-auer — COMMENTED** [review]
It would be great if you could check in a serialized payload from an actual PDB to test this implementation. Since we don't have the original source code, it's otherwise hard to verify that this implementation is correct. See [this test](https://github.com/willglynn/pdb/blob/7c1222e9ee7aba9218b95d0a5a3398a4ee3843d1/src/symbol/mod.rs#L1581-L1597) for an example.

**Comment by mahkoh** [comment]
I haven't encountered any 0x160a leafs so I've removed this code. I've added a test for 0x1609.

File: src/tpi/data.rs (hunk 1 above)
Comment (mahkoh) [reply src/tpi/data.rs:99]: I'm not aware of any such properties.

**Review by jan-auer — APPROVED** [review]

**Comment by jan-auer** [comment]
Thank you!

**Pull request merged**